staging/nvidia/charts/nvidia-device-plugin/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "nvidia-device-plugin.fullname" . }}-test-connection"
+  labels:
+{{ include "nvidia-device-plugin.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args:  ['{{ include "nvidia-device-plugin.fullname" . }}:{{ .Values.service.port }}']
+  restartPolicy: Never
```

Comment:
tests++

Reply:
I was following the test from gatekeeper, kube-oidc-proxy, and seems like some other chart (e.g., knative) does not come with a test yet. Could we add a test in a separate PR given the GPU functionality is tested with some CUDA image?

Reply:
Oh sorry, these comments are just meant as :+1: type comments :joy: